nucypher/cli/characters/felix.py

```diff
@@ -138,11 +138,11 @@ def felix(click_config,
                                                federated_only=felix_config.federated_only,
                                                network_domains=felix_config.domains,
                                                network_middleware=click_config.middleware)
-        
+
         # Add ETH Bootnode or Peer
         if enode:
             if geth:
-                felix_config.blockchain.interface.w3.geth.admin.addPeer(enode)
+                felix_config.blockchain.w3.geth.admin.addPeer(enode)
```

Comment:
@vepkenez @KPrasch Is `addPeer` a method we should add to the `blockchain.client`?